certbot/docs/contributing.rst

```diff
@@ -180,6 +180,9 @@ that can be used once the virtual environment is activated:
 - Press CTRL+C to stop this instance.
 - This instance is configured to validate challenges against certbot executed locally.
 
+.. note:: Some options are available to tweak the local acme server. You can execute
+    ``run_acme_server --help` to see the inline help of ``run_acme_server`` tool.
```

Comment:
The syntax here doesn't seem quite right. Rendered docs looks like:
![Screen Shot 2020-11-12 at 12 22 54 PM](https://user-images.githubusercontent.com/6504915/98992243-d4b7f800-24e1-11eb-9ff6-f50208e4632b.png)

Reply:
Fixed.